Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml

```
@@ -43,6 +43,10 @@
   - type: SwitchableEquipMode # Starlight
   - type: StaticPrice
     price: 70
+  - type: Tag
+    tags:
+      - WhitelistChameleon
+      - ScurretWearable # starlight
```

```suggestion
  # Starlight-start
  - type: Tag
    tags:
      - ScurretWearable
  # Starlight-end
```

and why you just adding this without any description/changelog?

---

oh crap, I think this got included accidentally - good catch

---

Should be fixed now.